sematic/resolvers/log_streamer.py

```diff
@@ -132,22 +148,16 @@ def _stream_logs_to_remote_from_file_descriptor(
         the remote prefix as arguments.
     """
     read_handle = os.fdopen(read_from_file_descriptor)
-
-    def do_exit(signal_num, frame):
```

Comment:
with the "End of Transmission" character, there was no longer a need to send a signal to stop the streamer